src/edu/csus/ecs/pc2/clics/API202306/JudgementService.java

```diff
@@ -0,0 +1,289 @@
+                                includeMaxRunTime = true;
+                            }
+                        }
+                        Set<String> exceptProps = new HashSet<String>();
+                        CLICSJudgement cJudgment = new CLICSJudgement(model, controller, run, exceptProps);
+                        // Remove max_run_time property if indicated
+                        if(!includeMaxRunTime) {
+                            exceptProps.add("max_run_time");
+                        }
+                        try {
+                            ObjectMapper mapper = JSONUtilities.getObjectMapper();
+                            // create filter to omit unused/bad properties (location, for example)
+                            SimpleBeanPropertyFilter filter = SimpleBeanPropertyFilter.serializeAllExcept(exceptProps);
+                            FilterProvider fp = new SimpleFilterProvider().addFilter("rtFilter", filter);
+                            String json = mapper.writer(fp).writeValueAsString(cJudgment);
+                            return Response.ok(json, MediaType.APPLICATION_JSON).build();
+                        } catch (Exception e) {
+                            return Response.status(Status.INTERNAL_SERVER_ERROR).entity("Error creating JSON for judgementId " + judgementId + " " + e.getMessage()).build();
+                        }
+                    }
+                }
+            }
+        }
+        return Response.status(Response.Status.NOT_FOUND).build();
+    }
+
+    /**
+     * Determine number of problems each team has solved
+     *
+     * @return a map of usernames to a map of problems solved
+     */
+    private HashMap<String,HashMap<ElementId,Boolean>> computeNumberSolvedMap() {
+        HashMap<String,HashMap<ElementId,Boolean>> solvedMap = new HashMap<String,HashMap<ElementId,Boolean>>();
+
+        String teamName;
+        ElementId probId;
+        HashMap<ElementId, Boolean> probMap;
+
+        // we have to go through each run and look for solved ones
+        for(Run run: model.getRuns()) {
+            if(run.isSolved()) {
+                teamName = run.getSubmitter().getName();
+                // see if we know about this team's solved problems
+                probMap = solvedMap.get(teamName);
+                if(probMap == null) {
+                    probMap = new HashMap<ElementId, Boolean>();
+                    solvedMap.put(teamName, probMap);
+                }
+                probId = run.getProblemId();
+                // if this team did not solve the problem yet, remember it did now
+                if(probMap.get(probId) == null) {
+                    probMap.put(probId, Boolean.valueOf(true));
+                }
+            }
+        }
+        return(solvedMap);
+    }
+
+    /**
+     * Retrieve access information about this endpoint for the supplied user's security context
+     *
+     * @param sc User's security information
```

Comment:
Same comment as on other files:  unused parameter `contest` should be documented.

Reply:
Fixed in next push.